session.go

```diff
@@ -285,6 +283,35 @@ func (s *session) GetGlobalSysVar(ctx context.Context, name string) (string, err
 	return value, nil
 }
 
+// GetGlobalStatusVar implements RestrictedSQLExecutor.GetGlobalStatusVar interface.
```

Comment:
implements GlobalVarAccessor.GetGlobalStatusVar interface.
So do the othe three functions.